scss/main/objects/_objects.date-picker.scss

```diff
@@ -105,6 +109,14 @@
             font-style: normal;
             color: $white-3;
         }
+
+        .lx-date-picker__opacity-full {
```

Comment:
It's a modifier, so regarding the BEM way, it should be `lx-date-picker--opacity-full`. I'll let @matmkian confirm it.